src/main/java/graphql/nadel/engine/OverallQueryTransformer.java

```diff
@@ -82,6 +82,7 @@
         long startTime = System.currentTimeMillis();
```

Comment:
As an aside - once we start pushing down 7 arguments to methods to hold execution state - we should have invested in a "state class" to make the methods easier.  Anway not for this PR but lesson for the future